xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -1029,6 +1029,11 @@ bool CDVDVideoCodecFFmpeg::GetPictureCommon(VideoPicture* pVideoPicture)
            (m_pCodecContext->profile == FF_PROFILE_VP9_2 ||
             m_pCodecContext->profile == FF_PROFILE_VP9_3))
     pVideoPicture->colorBits = 10;
+  else if (m_pCodecContext->codec_id == AV_CODEC_ID_AV1 &&
+           (m_pCodecContext->profile == FF_PROFILE_AV1_MAIN ||
```

Comment:
This is not correct because AV1 main profile can be 8 or 10 bits (not always 10). Same for high and professional.

See: https://en.wikipedia.org/wiki/AV1  profiles

Cannot use profile to determine bit depth

Reply:
also VP9 is specified in same wrong way
so what better way for now?

Reply:
then here you set the min value?
so i should remove all this change?

Reply:
Some time ago I tried to make an improvement here and it was not approved:
https://github.com/xbmc/xbmc/pull/19306

Reply:
i see it will require more development
so i remove it